TeamCode/src/main/java/org/firstinspires/ftc/teamcode/rework/Robot.java

```diff
@@ -68,14 +67,13 @@ public void update() {
                 try {
                     module.update();
                 }catch (Exception e){
-                    Log.d("Module", "Module couldn't update");
+                    Log.d("Module", "Module couldn't update: " + module.getClass().getSimpleName());

```

Comment:
Is there a way you can get the name without using reflection?

Reply:
@chop0 what's the problem with this? I think it's what you used for telemetry also

Reply:
yes, but im bad. reflection allegedly slows stuff down, but its fine ig